apps/hash-frontend/src/components/grid/grid.tsx

```diff
@@ -51,6 +51,14 @@ export type GridProps<T> = Omit<
   sortRows?: (rows: T, sort: ColumnSort<string>) => T;
 };
 
+const gridHeaderHeight = 42;
+
+export const gridHeaderHeightWithBorder = gridHeaderHeight + 1;
+
+export const gridRowHeight = 42;
+
+export const gridHorizontalScrollbarHeight = 17;
```

Comment:
This will vary across browsers, themes etc, but worst case scenario is that a table without enough data to fill the screen is not quite tall enough to fit the rows, causing an unnecessary vertical scrollbar, so I think it's fine to keep this assumption

Reply:
That's a fair point, I could increase it by a few pixels to preemptively account for that but that will create a gap between the last row and the scrollbar, so not sure it's a great trade-off.

Reply:
I don't think it's worth it. I'd expect entities tables will have enough rows to fill the max vertical height, in which case this number is irrelevant anyway.